app/helpers/file_set_helper.rb

```diff
@@ -15,12 +15,20 @@ def render_collection_list(fs)
   end
 
   def display_multiple(value)
-    auto_link(value.join(" | "))
+    # TODO: figure out why sometimes we get an string and some others an array
```

Comment:
This is a good TODO to leave in. I think the next few lines of code are a good candidate to take a "tell, don't ask" approach. Like so:

```ruby
string_value = Array(value).join(" | ")
auto_link(string_value)
```